lmfit/minimizer.py

```diff
@@ -1394,6 +1394,7 @@ def least_squares(self, params=None, **kws):
         except AbortFitException:
             pass
 
+        result.residual = self.__residual(ret.x)
```

Comment:
to keep the number of "real" function evaluations for the fit correct, we should probably add ```result.nfev -= 1``` as well